### PR TITLE
Added live clock and greeting based on time

### PR DESCRIPTION
**Pull request by maitrakhatri**
# features added
- Real time clock, updates every minute
- Greeting (morning, evening) based on time of the day

# working clock

### CODE REVIEW

File: gozo-browser-extension/src/index.js [hunk 1]
```diff
@@ -3,10 +3,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```
Comment (ranitasahaa1230) [inline gozo-browser-extension/src/index.js:5]: You can remove all these lines which are not needed ( here it's line no 5)

**Review by ranitasahaa1230 — APPROVED** [review]
UI is clean
Liked the folder structure

File: gozo-browser-extension/src/index.js (hunk 1 above)
Comment (maitrakhatri) [reply gozo-browser-extension/src/index.js:5]: yes I'll take care of it

**Review by Aditi0505 — APPROVED** [review]
Suggestion: You can create a folder for your context. You can later add the name of the user too for the message.
1. The UI looks really good.
2. Really liked the idea of displaying time and message accordingly.

**Comment by maitrakhatri** [comment]
> 

yes I'll take care of that

**Pull request merged**